apps/endatix-hub/features/public-form/ui/survey-component.tsx

```diff
@@ -97,16 +93,17 @@ export default function SurveyComponent({ definition, formId, submission }: Surv
     const processImages = async () => {
     
       for (let i = 0; i < options.question.value.length; i++) {
-        if (options.value[i].type.includes('image')) {
-            options.value[i].content = await resizeImage(
-              options.value[i].content,
-              options.value[i].type
+        let value = options.value[i];
+        if (value.type.includes('image')) {
+          value.content = await resizeImage(
+            value.content,
+            value.type
             );
         }
       }
     };
     
-    if(MAX_IMAGE_SIZE > 0) processImages();
+    if(MAX_IMAGE_SIZE > 0 && options.question.getType() == "file") processImages();
```

Comment:
This will solve the issue with null reference. I want to recommend couple of things (small ones)

1. Add null safety operator to `value.type.includes('image')` => `value?.type?.includes('image')` This way code will be safe
2. For debugging we try to avoid one line ifs and we would convert the 
`if(MAX_IMAGE_SIZE > 0 && options.question.getType() == "file") processImages();`
to 
```
if(MAX_IMAGE_SIZE > 0 && options.question.getType() == "file") {
   processImages();
}

We are consistent with this across both c# and ts

Reply:
Done